### PR TITLE
ci: 🎡 update dependencies, rules etc.

### PR DESCRIPTION
**Pull request by mariohamann**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,11 +68,44 @@ Since you're running `stylelint` natively, check out their documentation on how
...
+## Recommended settings
+### Stencil
+#### .eslintrc
+```js
+{
+  "extends": "./node_modules/@virtualidentityag/quality-gate/config/.eslintrc.js",
+  "rules": {
+    "class-methods-use-this": 0,
+    "no-script-url": 0,
+    "import/no-default-export": 1,
```
Comment (SheepFromHeaven) [inline README.md:85]: is already party of the quality gate

File: README.md [hunk 2]
```diff
@@ -68,11 +68,44 @@ Since you're running `stylelint` natively, check out their documentation on how
...
+    "react/prop-types": 0,
+    "@typescript-eslint/no-explicit-any": 0,
+    "react/jsx-key": 0,
```
Comment (SheepFromHeaven) [inline README.md:88]: why are we excluding key?

File: README.md [hunk 3]
```diff
@@ -68,11 +68,44 @@ Since you're running `stylelint` natively, check out their documentation on how
...
+#### .eslintrc
+```js
+{
+  "extends": "./node_modules/@virtualidentityag/quality-gate/config/.eslintrc.js",
+  "rules": {
+    "class-methods-use-this": 0,
+    "no-script-url": 0,
+    "import/no-default-export": 1,
+    "react/prop-types": 0,
+    "@typescript-eslint/no-explicit-any": 0,
```
Comment (SheepFromHeaven) [inline README.md:87]: I would not recommend this!

File: README.md [hunk 4]
```diff
@@ -68,11 +68,44 @@ Since you're running `stylelint` natively, check out their documentation on how
...
+    "react/jsx-key": 0,
+    "no-use-before-define": 0
+  },
+  "plugins": [
+    "eslint-plugin-local-rules"
```
Comment (SheepFromHeaven) [inline README.md:92]: if we include this here, we should explain it ^^

File: README.md [hunk 5]
```diff
@@ -68,11 +68,44 @@ Since you're running `stylelint` natively, check out their documentation on how
...
+}
+```
+#### .stylelintrc
+```js
+{
+  "extends": "./node_modules/@virtualidentityag/quality-gate/config/.stylelintrc.js",
+  "rules": {
+    "selector-class-pattern": ".",
+    "selector-no-qualifying-type": null,
+    "selector-type-no-unknown": [ true, { "ignore": ["custom-elements"] } ],
```
Comment (SheepFromHeaven) [inline README.md:103]: is party of quality gate

Comment (SheepFromHeaven) [reply README.md:103]: We (UI) should actually enable it again, to support with versioning

**Review by SheepFromHeaven — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (mariohamann) [reply README.md:87]: Yeah, I already wondered why we this in some repos. :-/

File: README.md (hunk 2 above)
Comment (mariohamann) [reply README.md:88]: This was the thing we experienced together where we would have had to set "keys" in arrays, but there weren't any arrays at all.

File: README.md (hunk 5 above)
Comment (mariohamann) [reply README.md:103]: Okay, removed it here.

**Review by SheepFromHeaven — APPROVED** [review]

**Pull request merged**